packages/fluentui/docs/src/examples/components/SplitButton/Usage/SplitButtonAlignMenuRightExample.shorthand.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { SplitButton } from '@fluentui/react-northstar';
+import { FilesGifIcon, FilesIllustratorIcon, FilesPdfIcon, FilesPhotoshopIcon } from '@fluentui/react-icons-northstar';
+
+const items = [
+  {
+    key: 'pdf',
+    content: 'Export to PDF',
+    icon: <FilesPdfIcon />,
+  },
+  {
+    key: 'pds',
+    content: 'Export to PDS',
+    icon: <FilesPhotoshopIcon />,
+  },
+  {
+    key: 'gif',
+    content: 'Export as GIF',
+    icon: <FilesGifIcon />,
+    disabled: true,
+  },
+  {
+    key: 'eps',
+    content: 'Export to EPS',
+    icon: <FilesIllustratorIcon />,
+  },
+];
+
+const SplitButtonAlignMenuRightExampleShorthand = () => (
+  <>
+    <SplitButton
+      menu={items}
+      button={{
+        content: items[0].content,
+        key: items[0].key,
+        icon: items[0].icon,
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message-icon-content',
+      }}
+      onMainButtonClick={() => alert('button was clicked')}
```

Comment:
Let's avoid `alert` if it's not necessary

 ```suggestion
```

Reply:
Done